src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -83,3 +65,20 @@ new Replay({
   // any further configuration here is passed directly to rrweb
 });
 ```
+
+## Start and Stop Recording Manually
```

Comment:
I'm assuming here we prefer users keep with defaults and let the SDK start/stop. Is that valid? For that reason, moving to the bottom.

These APIs can become a source of support issues, where the SDK is already start/stopping but users decide to call it too and then are confused by the duration of replays. For that reason, if `debug=true` and `start()` is called but there's already a replay running, worth logging out, for discoverability sake.

Do we know of some use cases when users want to start/stop manually? Is it worth adding here to the docs "you for example want to xyz"?